src/DbToolsBundle.php

```diff
@@ -4,15 +4,27 @@
 
 namespace MakinaCorpus\DbToolsBundle;
 
-use MakinaCorpus\DbToolsBundle\DependencyInjection\Compiler\DbToolsPass;
+use MakinaCorpus\DbToolsBundle\Bridge\Symfony\DbToolsBundle as TheRealBundle;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\HttpKernel\Bundle\Bundle;
 
-class DbToolsBundle extends Bundle
+/**
+ * @deprecated
+ *   This class is deprecated and will be remove in next major.
+ * @see \MakinaCorpus\DbToolsBundle\Bridge\Symfony\DbToolsBundle
+ */
+class DbToolsBundle extends TheRealBundle
 {
     #[\Override]
     public function build(ContainerBuilder $container): void
     {
-        $container->addCompilerPass(new DbToolsPass());
+        \trigger_deprecation(
```

Comment:
This reminds me that we have to test how the PHAR compiled tool will behave when used in PHP 8.1 (we must ensure that various polyfills are integrated into the PHAR file).